Create team-manual.md (#3028)
alan-turing-institute/the-turing-way: book/website/collaboration/team-manual.md

(cl-team-manual-benefits)=
## Benefits of Team Manuals

- Clarity about expectations
- Everyone has access to the same information
- Everyone can contribute to the team manual and updating will take less time
- Discussions around ways of working are normalised
- Group members will think more actively about how to do research
- Answers to frequently asked questions will be available (knowledge management)
- Can contribute to a safer lab environment (technically and socially)
- You can hold people more easily accountable (see below)

(cl-team-manual-content)=
## What to include in a Team Manual? 

### Main points 
1. Mission/vision statement:
What are the research goals of the team, why are these the goals? 
How does the team work? What is the history of the group? 
How does the team fit into the department/institute? 
What are the [principles](https://www.lesleahlusko.org/principles-of-community-for-our-intellectual-family/)? 
How do we do science in a way that fits with our mission/vision statement. 
2. Roles and expectations: What are the responsbilities of each team member, including the PI? 
When will these roles be discussed when a new member joins the team?

### More details

> - How is the lab manual updated? 

Reviewer: ```suggestion
- Meta-processes: How is the lab manual updated? 
```

> Who can make propose the changes (ideally every member) and what is the process? 

Reviewer: I suspect one of those verbs is duplicated, here's a proposed fix:

```suggestion
Who can propose the changes (ideally every member) and what is the process? 
```

Do you keep multiple versions of the lab manual? 
Where will the lab manual be made available for all lab members to access?
- Open Science: 
How does the team manage and share code, data, materials? 
How is documentation done? 

> - Lab manual: 

Reviewer: Each bullet point has an introductory term such as "Communication", "Behaviour", etc - this introductory term is "Lab manual". Given that the entire chapter is on "Team Manuals", and that the content of this bullet point is on internal research processes, maybe we can rename this introductory term to something like this?

```suggestion
- Research process: 
```

how is research conducted at the lab? 
How is the data analysed? 
How is the code written, version controlled? 
See for example the [Research toolkit by the Open Data Institute](https://theodi.org/article/check-out-our-rd-open-research-tools/). 
Is work of lab members reproduced by other lab members (for example, to learn new techniques and to validate previous work)?
- Communication: 
How do people engage with each other? 
- Under what condition can people provide their best work? 
What support is needed? 
How do people work? 
Can they work remotely? 
What are people's preferred working hours? 
What to arrange when you go on a holiday? 
Are there any weekly/daily tasks? 
- Inclusivity: 
How are hiring practices inclusive of minoritised groups? 
- Resources: 

> anything that lab members need access to (servers, software, tools, room keys, library cards etc)

Reviewer: etc is flagged by the latin checker - what do you usually turn this into?

```suggestion
anything that lab members need access to (servers, software, tools, room keys, library cards, and the like)
```

- On/off boarding: 
A list of what new members need to do when they start out, and a list what leaving members need to do before they go *(ref to on/off boarding)*. 
- Behaviour: 
Consider setting up a code of conduct or policy on scientific integrity (and align with the institutional/national policies on this - see below). 
What should people do when they encounter harassment/discrimination? What is the lab culture? 

> - How to provide appropriate credit to the right people? 

Reviewer: ```suggestion
- Credit: How to provide appropriate credit to the right people? 
```

How to ensure that each contribution is recognised and rewarded? 
This includes (lab) maintenance work, discussions, and ideas. 

> - When is a lab member successful? 

Reviewer: ```suggestion
- Metrics: When is a lab member successful? 
```

How do you discuss and measure this progress so that the lab member can more easily succeed? 
How are successes celebrated? 
How are reproducible, ethical and inclusive data science practices included in this assessment? 
How is maintenance work evaluated and celebrated? 
- How are mistakes handled (see [Rouder et al. 2019](https://doi.org/10.1177/25152459188019))? 
How is insecurity being normalised? How is feedback given and acted upon?
- Engagement: 
what are the expectations around public outreach? 
What is the social media policy for personal accounts? 

> How are all the materials made accessible (image descriptions, translations, etc (ref to chapter?) 

Reviewer: etc is flagged by the latin checker - what do you usually turn this into? Also, this is a TODO to link the chapter you had in mind when writing this.

```suggestion
How are all the materials made accessible (image descriptions, translations, and the like (ref to chapter?) 
```

How is the lab involved in the department? How does the lab work with participants?

> - Publications. What needs to happen before a paper is published? what are the preprint policies? 

Reviewer: ```suggestion
- Publications: What needs to happen before a paper is published? what are the preprint policies? 
```

Guidelines for PhD candidates could be at least 1-2 local/regional conferences during the first year(s) of PhD, and 1 major international one later in the project.
- Finances: What grants pay for the lab? 
What will/won't the lab pay for? 
What grants can people apply for? 
What to expect in terms of reimbursements? 
- Grants: How to deal with awarded grants? 
Should they be public to the whole lab or should only summaries and outcomes be available? 
Who can apply to grants?
- Mentorship and development: 
How do you get feedback from others? 
How do you collaborate in/outside of the lab? 
What are the opportunities for development and mentoring. 
How do you discuss career development? 
See [Mentoring and Advising Contract Points of Agreement](https://docs.google.com/document/d/1oOPjjF8RJ1EOFvZMFAKe19hUVyQ5FRpS-tPV_uE1dcI/edit) or a [career template plan](https://www.skillsweaving.com/Downloads) as an example. How can you get mentoring outside of the lab?
- Training: How are training opportunities formalised? 
What is the miminum amount of personal development days per year? 
What is the budget available? 

> - Where to go for help? 

Reviewer: ```suggestion
- In case of crisis: Where to go for help? 
```

This can be support in/outside the lab, schedules of trainings, and contact persons. 

> - Reading list that lab members should read.

Reviewer: ```suggestion
- Scientific resources: Reading list that lab members should read.
```

> - How do all lab members hold each other accountable? What are procedures to follow? 

Reviewer: ```suggestion
- Accountability: How do all lab members hold each other accountable? What are procedures to follow? 
```

How to ensure that this accountability is not on individuals but the collective? (see also below)

### Institutional / national guidelines that affect Team Manuals
Institional and national policies and guidelines prevail over the Lab Manual. 
- Ethics and Safety: What are the safety procedures? What are the ethic procedures? 
Is there a research integrity office? 
- Research Integrity and culture: Is there an institutional policy on this? 
If there is a dispute, what would be the procedure? Can you contact Human Resources about this?

> - What are the guidelines for research with animals, humans, personal data etc? (see chapter ...)

Reviewer: adding a comment here to highlight this as a TODO. Maybe the entire Guide for Ethical research?

```suggestion
- What are the guidelines for research with animals, humans, personal data etc? (see chapter ...)
```

Reviewer: Also, etc is flagged by the latin checker - what do you usually turn this into?

- Where is institutional support situated? (Human Resources, Integrity Office, Research Data Management, Education)
 
### Collaboration and consortium guidelines that extend beyond the Team's Manual
- How do we choose who to collaborate with? 
- How do we work together between teams? 
How do we ensure the manuals are not conflicting? 
Which Team Manual has precidence?

(cl-team-manual-start)=
## How to set up and start a Team Manual?
Ideally the group leader or another senior member of the group will start setting up a Team Manual. 
They are in a better position to do so and have power over decision making. 
This does not mean only senior members can be involved: the whole group should be able to provide input and it is important to hold discussions over points where there is no agreement in the group. 

If you want to convince your supervisor that having a Team Manual is important: 
- Check if a comparable lab is using a Lab Manual and show them this. 

> You can also use the list provided (see below [A list of Lab Manuals]) as an example. 

Reviewer: Another comment to highlight the TODO to add the internal link here

```suggestion
You can also use the list provided (see below [A list of Lab Manuals]) as an example. 
```

 
(cl-team-manual-accountable)=
## How do you keep each other accountable? 

When a Team Manual is established it is important that everyone is on board.  
If someone is not following the Lab Manual it needs to be discussed why this is the case. 
It could be that the Lab Manual needs to be updated, or a discussion needs to take place to re-align the values of the group members, or perhaps this lab member is not a right fit for the group. 

If someone breaks the agreements it needs to be clear what the consequences are. 
The Team/Lab needs to be held accountable, as otherwise there is no point of having a Lab Manual. 
See also [Patrick Lencioni's 'Teamwork: The Five Dysfunctions of a Team'](https://www.worldcat.org/title/794274430): 
- Absence of trust
- Fear of Conflict
- Lack of commitment
- Avoidance of accountability
- Inattention to results

(cl-team-manual-assess)=
## How to assess your lab culture?
* Leslie Vosshall's [lab survey](https://docs.google.com/forms/d/e/1FAIpQLScGCi7iACgmVBhFcE7G90oPwuTs-g9CQkrDmOUoQ4FvoT9CfA/viewform) to measure whether your lab is happy 

> * [Hernandez et al. 2021](https://doi.org/10.1101/2021.12.08.471870 )

Reviewer: I think the title in addition to the authors can help readers assess whether they want to click on the link

```suggestion
* [Hernandez et al. (2021). Improving Lab Culture through Self-Assessment: A Case Study](https://doi.org/10.1101/2021.12.08.471870 )
```

(cl-team-manual-examples)=
## Examples of Team Manuals

### Copy of a Team Manual template
* [Team Manual Template](https://docs.google.com/document/d/1LqGdtHg0dMbj9lsCnC1QOoWzIsnSNRTSek6i3Kls2Ik/edit)

### Examples of different types of Team/Lab Manuals

> * [Code of Conduct Whitaker lab](https://github.com/WhitakerLab/Onboarding/blob/master/CODE_OF_CONDUCT.md) and the [Basin Research Group](https://docs.google.com/document/d/10y7TP48ik1rcQBPA5Do8mZM7DJ5EbF0hyWP-csgb1QE/edit)

Reviewer: The inclusive language warning is somewhat of a a "fluke" and can be ignored in my opinion - It flagged a Git ``master`` branch in a link (so in text that will not be rendered).
